packages/start/fs-router/fileRoutesImport.ts

```diff
@@ -1,7 +1,21 @@
-export default function fileRoutesImport({ types: t }) {
+// @ts-ignore
```

Comment:
Why `@ts-ignore` has been added here?

Reply:
Otherwise it'll warn:

Cannot find module '@babel/types/lib' or its corresponding type declarations.ts(2307)

Reply:
Doesn't `[@types/babel__types](https://www.npmjs.com/package/@types/babel-types)` have the types?

Reply:
I'll have another look at this